fuel-core/src/executor.rs

```diff
@@ -268,6 +270,21 @@ impl Executor {
         Ok(())
     }
 
+    /// Verify all the predicates of a tx.
+    pub fn verify_tx_predicates(&self, tx: &Transaction) -> Result<(), Error> {
+        // fail if tx contains any predicates when predicates are disabled
+        if !self.config.predicates {
```

Comment:
If the predicates verification is disabled in the config, shouldn't we ignore them?

Reply:
I wanted to play it safe and just outright reject txs that include predicates, that way someone can't spend invalid predicates in our stable environment